backend/app/processing/vanna_integrator.py

```
@@ -0,0 +1,267 @@
+            # Set the chromadb_path in the config
+            chatbot_config["chromadb_path"] = chatbot_dir
+            
+            # Create the instance with the modified config
+            vn = MyVanna(config=chatbot_config)
+            self._instance[chatbot_id] = vn
+            
+            logger.info(f"Created Vanna instance for chatbot {chatbot_id} with storage at {chatbot_dir}")
+        
+        return self._instance[chatbot_id]
+
+    def process_sqlite(self, config: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)